dashboard/dashboard.go

```diff
@@ -27,13 +27,17 @@ package dashboard
 
 import (
 	"fmt"
+	"io"
 	"net"
 	"net/http"
 	"sync"
 	"sync/atomic"
-	"time"
 
-	"io"
+	"github.com/ethereum/go-ethereum/eth"
+	"github.com/ethereum/go-ethereum/les"
+
```

Comment:
Pls remove empty lines from in between these imports. The only empty line should be between stdlib and externals.